jdbc-handler/src/main/java/org/apache/hive/storage/jdbc/dao/GenericJdbcDatabaseAccessor.java

```diff
@@ -172,6 +177,67 @@ public int getTotalNumberOfRecords(Configuration conf) throws HiveJdbcDatabaseAc
     }
   }
 
+  public RecordWriter getRecordWriter(TaskAttemptContext context)
+          throws IOException {
+    Configuration conf = context.getConfiguration();
```

Comment:
L182-185 can be moved below first if condition.

Reply:
> L182-185 can be moved below first if condition.

Moved.